src/components/new-request/search-components/business-lookup-fetch.vue

```diff
@@ -22,6 +22,7 @@
         :hint="hint"
         @click:append="onBusiness(null)"
         @keyup.delete="onBusiness(null)"
+        @keydown.enter.native.prevent
```

Comment:
It might also be possible to tell the form to prevent submitting itself (`v-form @submit.prevent`) but this looks good, too.